### PR TITLE
[External]Add OpenACC Validation and Verification testsuite

### PR DESCRIPTION
**Pull request by wangxin0321**
OpenACC is a high-level directive-based parallel programming model that can manage the sophistication of heterogeneity in architectures and abstract it from the users. The portability of the model across CPUs and accelerators has gained the model a wide variety of users. This means it is also crucial to analyze the reliability of the compilers' implementations.

To address this challenge, the OpenACC Validation and Verification team introduces a validation testsuite ，supported by a streamlined infrastructure, to verify the OpenACC implementations across various compilers and system architectures.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -241,6 +241,12 @@ include(TestSuite)
 include(SingleMultiSource)
 # Needs by External/sollve_vv.
 find_package(OpenMP)
+# Needs by External/OpenACC_VV
```
Comment (yuanfang-chen) [inline CMakeLists.txt:244]: Is it possible to move this check to `External/OpenACC_VV/CMakeLists.txt`?

File: External/OpenACC_VV/README [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+   -DCMAKE_C_COMPILER=${HOME}/install/llvm/bin/clang                   \
+   -DCMAKE_CXX_COMPILER=${HOME}/install/llvm/bin/clang++               \
+   -DCMAKE_Fortran_COMPILER=${HOME}/install/llvm/bin/flang             \
+   -DTEST_SUITE_OpenACC_OFFLOADING_FLAGS="-fopenacc;-lm;-foffload='-lm';"  \
+   -DTEST_SUITE_OpenACC_OFFLOADING_LDFLAGS="-fopenacc;-lm;-foffload='-lm';" \
+   ../ 
+
+To make:
+$ LD_LIBRARY_PATH=${HOME}/install/llvm-project/release/lib 
+$ ninja -k 0 check
```
Comment (yuanfang-chen) [inline External/OpenACC_VV/README:44]: `ninja check`

**Review by yuanfang-chen — CHANGES_REQUESTED** [review]
Make sure all indentations are two spaces.

**Comment by wangxin0321** [comment]
Have been repaired

**Review by yuanfang-chen — APPROVED** [review]
LGTM

**Pull request merged**